recipe/bld.bat

```diff
@@ -0,0 +1,12 @@
+copy "%RECIPE_DIR%\build.sh" .
+set PREFIX=%PREFIX:\=/%
+set SRC_DIR=%SRC_DIR:\=/%
+set MSYSTEM=MINGW%ARCH%
+set MSYS2_PATH_TYPE=inherit
+set CHERE_INVOKING=1
+bash -lc "./build.sh"
+if errorlevel 1 exit 1
+
+move %PREFIX%\bin\*.exe %LIBRARY_BIN% || exit 1
```

Comment:
We may want to move the headers and library to "Windows" directories too, but then we may need to patch the `gridutils` makefile... Not sure what to do yet.